packages/nimble-components/src/table/components/row/index.ts

```diff
@@ -133,6 +133,12 @@ export class TableRow<
     @observable
     public animationClass = '';
 
+    /**
+     * @internal
+     */
+    @attr({ attribute: 'allow-hover-styling', mode: 'boolean' })
```

Comment:
nit: Including `styling` in the name feels weird / too leaky about the implementation. Can we do `allow-hover`?